scripts/util.lua

```diff
@@ -630,5 +630,24 @@ function module.replaceupvalue(func, name, replacement)
 	end
 end
 
+
+--- Format string
+--- With supports for "$" replace-able
+-- @str string to replace
+-- @param replacing fields
+-- @return replaced string
+local _StringFormat = string.format
+string.format = function(str, ...)
```

Comment:
Replacing built-in methods is never advisable, and I do not want to do so. See pull request comment.